platform/ui-next/src/components/DataRow/DataRow.tsx

```diff
@@ -178,25 +144,20 @@ const DataRow: React.FC<DataRowProps> = ({
           className="max-w-md"
         >
           <div className="text-secondary-foreground flex flex-col space-y-1 text-sm leading-normal">
-            {details.map((line, lineIndex) =>
-              renderDetailText(line, line.startsWith('  ') ? 1 : 0)
-            )}
+            {details.map(line => renderDetailText(line, line.startsWith('  ') ? 1 : 0))}
           </div>
         </TooltipContent>
       </Tooltip>
     );
   };
 
   return (
-    <div
-      ref={rowRef}
-      className={`flex flex-col ${isVisible ? '' : 'opacity-60'}`}
-    >
+    <div className={`flex flex-col ${isVisible ? '' : 'opacity-60'}`}>
       <div
         className={`flex items-center ${
           isSelected ? 'bg-popover' : 'bg-muted'
         } group relative cursor-pointer`}
-        onClick={onSelect}
+        onClick={e => onAction(e, 'jumpToMeasurement')}
```

Comment:
I don't think these changes make sense. `DataRow` is a general row-wise visualization that we use for measurements *and* segmentations, and potentially contours in the future. So, it really shouldn't mention `jumpToMeasurement` or `toggleVisibilityMeasurement`, or anything like that.

Also you have removed the `rowRef` which we need